active/RadioMenuFlyoutItem/RadioMenuFlyoutItem.md

```diff
@@ -0,0 +1,61 @@
+# Description
+A radio button menu flyout item is a menu item that is mutually exclusive with other menu items in its group. 
+This control contol allows apps to present users with menus containing multiple options, where a user would choose only of these options at a given time.
+
+
+# Examples
+
+Add two or more *RadioMenuFlyoutItems* to the content of *MenuBarItem*, *MenuFlyout*, or *MenuFlyoutSubItem*.
+
+![Three radio menu flyout items in a View goup that allow a user to select the size of icons](RadioMenuFlyoutItems.png)
+
+````Xaml
+<MenuFlyoutSubItem Text="View">
+    <muxc:RadioMenuFlyoutItem Text="Small icons"/>
+    <muxc:RadioMenuFlyoutItem Text="Medium icons" IsChecked="True"/>
+    <muxc:RadioMenuFlyoutItem Text="Large icons"/>
+</MenuFlyoutSubItem>
+````
+
+
+For multiple groupings of radio buttons within a single menu, be sure to specify the `GroupName` property of each set.
```

Comment:
Should explain first what the general grouping rule is, then talk about the example.

Speaking of which, why is GroupName necessary here?  This sample here with RadioButton doesn't require GroupName.  Are we special-casing the separator for some reason?

Reply:
@MikeHillberg - the example shows two groups. If I didn't specify a GroupName, all five items (two above and three bellow the separator) would be treated as a single group.

Reply:
Sorry, good example.